content/fundamentals/advanced.md

```diff
@@ -0,0 +1,93 @@
+### Advanced Concepts
+
+So far we've breezed over a couple of advanced topics in Typescript like decorators and how the DI system is actually working under the hood. If all you want is a system that works, ou can probably move on to another section, but if you want to get a deeper understanding of how Nest works, this is the section for you.
+
+#### The @Injectable() Decorator
```

Comment:
```suggestion
#### The `@Injectable()` Decorator
```

Reply:
I think I'd rather keep this without the \`\` wrapping the decorator. I don't know anywhere else in the docs we have it being wrapped and part of a heading, but this, I think, is also a first for having it as part of a heading.

@kamilmysliwiec what do you think here?